Makefile

```diff
@@ -15,10 +15,10 @@ ci-build-misc: print-go-version ci-update-tools proto bins shell-check copyright
 clean: clean-bins clean-test-results
 
 # Recompile proto files.
-proto: clean-proto buf-lint api-linter protoc goimports-proto proto-mocks copyright-proto
+proto: clean-proto buf-lint api-linter protoc proto-clients goimports-proto proto-mocks copyright-proto
 
 # Update proto submodule from remote and recompile proto files.
-update-proto: clean-proto update-proto-submodule buf-lint api-linter protoc update-go-api goimports-proto proto-mocks copyright-proto gomodtidy
+update-proto: update-proto-submodule proto gomodtidy
```

Comment:
drive-by fix: `update-proto` can re-use the `proto` target

Reply:
In theory none of these are correct usage of make. Makefiles shouldn't be written assuming that dependencies will be "built" in any particular order. But I think as long as all of them are declared in `.PHONY` (you should add the ones that are missing) and no one uses `-j` then it'll continue to work.